actors/verifreg/src/lib.rs

```diff
@@ -767,6 +919,18 @@ where
     )
 }
 
+fn can_claim_alloc(claim_alloc: &SectorAllocationClaim, alloc: &Allocation, curr_epoch: ChainEpoch) -> bool {
+    let sector_lifetime = claim_alloc.sector_expiry - curr_epoch;
+
+    claim_alloc.provider == alloc.provider
+    && claim_alloc.client == alloc.client 
+    && claim_alloc.piece_cid == alloc.data
+    && claim_alloc.piece_size == alloc.size 
+    && curr_epoch <= alloc.expiration
```

Comment:
This `<=` matches what I wrote in the FIP draft, but am now thinking the name should imply `<`, "before" semantics.